src/SmartComponents/Remediations/RemediationWizard.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import * as api from '../../api/remediations';
+import { Wizard } from '../../PresentationalComponents/Wizard';
+import Deferred from '../../Utilities/Deferred';
+
+import NameStep from './steps/Name';
+
+import './RemediationWizard.scss';
+
+let mountedInstance = false;
+
+export function getMountedInstance () {
+    return mountedInstance;
+}
+
+function createRemediation (name, { issues, systems }) {
+    name = (name && name.length) ? name : 'Unnamed remediation';
+
+    return api.createRemediation({
+        name,
+        add: {
+            issues,
+            systems
+        }
+    });
+}
+
+function createNotification (remediation) {
+    return {
+        variant: 'success',
+        title: 'Remediation created',
+        description: `Remediation ${remediation.name} has been created`,
+        dismissDelay: 8000
+    };
+}
+
+class RemediationWizard extends Component {
+
+    constructor (props, ctx) {
+        super(props, ctx);
+        this.store = ctx.store;
+        this.state = {
+            open: false
+        };
+
+        [ 'setOpen', 'openWizard', 'closeWizard' ].forEach(fn => this[fn] = this[fn].bind(this));
```

Comment:
Can you please add `babel-plugin-transform-class-properties` dependency and add it to `.babelrc.js`? We will no longer need to use bind and just
```JS
setOpen = (open) => {}
openWizard = (data) => {}
closeWizard = (submitted) => {}
```